test/Dockerfile

```diff
@@ -12,45 +12,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM debian:8.5
+FROM gcr.io/kubernetes-charts-ci/chart-testing:v1.0.0
```

Comment:
Where is the source for this FROM? It's under our control which is why I'm curious.

Side note, I'm starting to ponder Quay because of the container image security scanning.

Reply:
Well, that's the image containing the new factored out testing stuff.